plugin-gradle/README.md

```diff
@@ -468,8 +468,28 @@ spotless {
 }
 ```
 
+<a name="prettier-plugins"></a>
+### Using community plugins for prettier
+
+Since spotless uses the actual npm prettier package behind the scenes, it is possible to use prettier with
+community-plugins in order to support even more file types.
```

Comment:
Maybe link to somewhere where people can find community plugins?

Reply:
I see two options:

- https://www.npmjs.com/search?q=prettier-plugin
- https://prettier.io/docs/en/plugins.html#official-plugins

The first one opens up the whole universe. The second one is probably a bit dated, but seems more official...

Reply:
ok, added the links